lib/Service/LoginService.php

```diff
@@ -253,6 +260,22 @@ public function login($profile, $userSession, $request)
                 }
             }
 
+            if ($this->config->getSystemValue('oidc_login_update_avatar', false) && $profile[$attr['photoURL']]) {
+                try {
+                    $curl = curl_init($profile[$attr['photoURL']]);
+                    curl_setopt($curl, CURLOPT_HEADER, false);
+                    curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+                    curl_setopt($curl, CURLOPT_BINARYTRANSFER, true);
+                    curl_setopt($curl, CURLOPT_USERAGENT, 'Mozilla/5.0');
```

Comment:
I'm wondering if there is a specific reason for why you chose to set this specific user agent?

Reply:
I have read that some webservers may expect user-agent to be set. I am not sure this is very important, I can remove this if you think this is useless.

Reply:
Ok. This is done.